pkg/datarepair/checker/config.go

```diff
@@ -38,19 +38,22 @@ func (c Config) initialize(ctx context.Context) (Checker, error) {
 		return nil, Error.New("unable to get master db instance")
 	}
 
-	db, ok := ctx.Value("masterdb").(interface {
+	irdb, ok := ctx.Value("masterdb").(interface {
 		Irreparable() irreparable.DB
 	})
 	if !ok {
 		return nil, Error.New("unable to get master db instance")
 	}
-	o := overlay.LoadServerFromContext(ctx)
-	redisQ, err := redis.NewQueueFrom(c.QueueAddress)
-	if err != nil {
-		return nil, Error.Wrap(err)
+	qdb, ok := ctx.Value("masterdb").(interface {
```

Comment:
This shouldn't be separate. Take a look at https://github.com/storj/storj/blob/master/pkg/overlay/config.go#L71